efscripts.txt

```diff
@@ -1,6 +1,6 @@
 
 -- RUN THIS FROM THE WEB PROJECT FOLDER
-dotnet ef migrations add DiscordUsername --context appdbcontext -p ../DevBetterWeb.Infrastructure/DevBetterWeb.Infrastructure.csproj -s DevBetterWeb.Web.csproj -o Data/Migrations
+dotnet ef migrations add InviteEntityActiveBool --context appdbcontext -p ../DevBetterWeb.Infrastructure/DevBetterWeb.Infrastructure.csproj -s DevBetterWeb.Web.csproj -o Data/Migrations
```

Comment:
We don't need to update this every time we create a new migration - it's meant to be a template with the name as a placeholder to be replaced (in the console, not here).